p2p/src/authenticated/actors/router/ingress.rs

```diff
@@ -9,15 +9,17 @@ use futures::{
     SinkExt,
 };
 
+/// Messages that can be processed by the router.
 pub enum Message<P: PublicKey> {
+    /// Notify the router that a peer is ready to communicate.
     Ready {
         peer: P,
         relay: peer::Relay,
         channels: oneshot::Sender<Channels<P>>,
     },
-    Release {
-        peer: P,
-    },
+    /// Notify the router that a peer is no longer available.
+    Release { peer: P },
```

Comment:
I suppose we could make this `Release(P)` but like the clarity that naming the fields provides.